server/src/main/scala/services/graphstages/Crunch.scala

```diff
@@ -107,11 +124,11 @@ object Crunch {
 
     toUpdate.map(_.apiFlight).foreach(ua => {
       oldFlightsById.get(ua.uniqueId).map(_.apiFlight).foreach(oa => {
-        if (ua.Estimated != oa.Estimated) log.info(s"${ua.IATA} changed estimated ${oa.Estimated} -> ${ua.Estimated}")
-        if (ua.Actual != oa.Actual) log.info(s"${ua.IATA} changed touchdown ${oa.Actual} -> ${ua.Actual}")
-        if (ua.EstimatedChox != oa.EstimatedChox) log.info(s"${ua.IATA} changed estchox   ${oa.EstimatedChox} -> ${ua.EstimatedChox}")
-        if (ua.ActualChox != oa.ActualChox) log.info(s"${ua.IATA} changed actchox   ${oa.ActualChox} -> ${ua.ActualChox}")
-        if (ua.ActPax != oa.ActPax) log.info(s"${ua.IATA} changed actpax   ${oa.ActPax} -> ${ua.ActPax}")
+        if (ua.Estimated != oa.Estimated) log.debug(s"${ua.IATA} changed estimated ${oa.Estimated} -> ${ua.Estimated}")
```

Comment:
do you wanna leave these logs in?

Reply:
possibly not. i demoted them to debug, but it's probably fine to remove them i think